mobly/utils.py

```diff
@@ -474,36 +474,23 @@ def run_command(
       shell=shell,
       cwd=cwd,
       env=env,
-      universal_newlines=universal_newlines,
+      text=universal_newlines,  # "text" is introdcued in Python 3.7.
```

Comment:
is `text` functionally identical to `universal_newline`?

Reply:
Yes, `text` is the modern and preferred keyword for handling text mode. `universal_newlines` is the older and deprecated equivalent.